addons/containerd/1.5.10/install.sh

```diff
@@ -84,6 +84,28 @@ function containerd_install() {
     fi
 }
 
+function conatinerd_host_init() {
+    containerd_install_libzstd_if_missing
+}
+
+function containerd_install_libzstd_if_missing() {
+    local src="$DIR/addons/containerd/$CONTAINERD_VERSION"
+
+    if yum list installed libzstd >/dev/null 2>&1 ; then
+        return
+    fi
```

Comment:
I feel like this should go AFTER the lsb_dist check
otherwise won't it break for ubuntu?

Reply:
fixed